utilities/vrt_helper.py

```
@@ -63,9 +57,7 @@
             TestRunStatus.OK.name,
         )
 
-    def shoot_page_ang_ignore_elements(
-        self, baseline_name: str, elements: list[WebElement]
-    ):
+    def shoot_page_ang_ignore_elements(self, baseline_name: str, elements: list[WebElement]):
```

Ruff (0.8.2)

Missing return type annotation for public function `shoot_page_ang_ignore_elements`

Add return type annotation: `None`

(ANN201)